crates/sui-graphql-rpc/examples/stake_connection/stake_connection.graphql

```diff
@@ -1,21 +1,32 @@
-{
+query stake_connection {
```

Comment:
this creates a fragment or something right

Reply:
Not sure I understood your question, but if I understand your question right, yes, this is just the name of the query. By giving it a name, you can select to run specific queries in the IDE.